extensions/amp-recaptcha-input/amp-recaptcha-input.md

```diff
@@ -13,3 +13,82 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+
+# <a name="`amp-recaptcha-input`"></a> `amp-recaptcha-input`
+
+<table>
+<tr>
+<td width="40%"><strong>Description</strong></td>
+<td>Appends a <a href="https://developers.google.com/recaptcha/docs/v3">reCAPTCHA v3</a> token to <a href="https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md">AMP form</a> submissions.</td>
+</tr>
+<tr>
+<td width="40%"><strong>Status</strong></td>
+<td>Experimental. You must turn on the `amp-recaptcha-input` experiment to use this component.</td>
+</tr>
+<tr>
+<td width="40%"><strong>Required Script</strong></td>
+<td><code>&lt;script async custom-element="amp-recaptcha-input" src="https://cdn.ampproject.org/v0/amp-recaptcha-input-0.1.js">&lt;/script></code></td>
+</tr>
+<tr>
+<td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+<td>nodisplay</td>
+</tr>
+<tr>
+<td width="40%"><strong>Examples</strong></td>
+<td><a href="https://github.com/ampproject/amphtml/blob/master/examples/recaptcha.amp.html">Development Example</a></td>
+</tr>
+</table>
+
+[TOC]
+
+## Behavior
+
+This extension adds a parameter containing a recaptcha response token when a parent `<form>` element submits. `amp-recaptcha-input` does this by creating an iframe to load the reCAPTCHA v3 api script using the provided site key, and calling greaptcha.execute with the provided site key and action.
```

Comment:
I think `greaptcha.execute` was intended to be `grecaptcha.execute`? And can you wrap it in ``` please :)

Reply:
Done 😄 I went ahead and wrapped all `grecaptcha` in backticks as well 😄